### DISCUSSION TITLE
Edit Translation

### DISCUSSION THREAD

**Discussion by Stoufiler** [Thomas55555/husqvarna_automower#204] [Q&A]
Hi !

How can I edit "nettoyage" to "tonte" ?

I tried to search in all this repo but can't find it 🧐

![image](https://user-images.githubusercontent.com/44873931/164192495-402860d3-99cf-418a-89eb-0d67285eac14.jpeg)

**Comment by Thomas55555** [comment]
Should actually be translated already:
https://github.com/Thomas55555/husqvarna_automower/blob/474ba1622efc8c7b043c8c29c3d6601fada1a980/custom_components/husqvarna_automower/translations/vacuum.fr.json

**Comment by Stoufiler** [comment]
Ok, now I understand why it written "Nettoyage".

After reading code, I saw that the integration use VACUUM entity so we can't edit translation

Of course in dashboard of entity we see translations presents in https://github.com/Thomas55555/husqvarna_automower/blob/474ba1622efc8c7b043c8c29c3d6601fada1a980/custom_components/husqvarna_automower/translations/vacuum.fr.json
